src/configure_edison

```diff
@@ -860,42 +887,42 @@ def do_name_form():
 def wired_form():
     global Settings
     Host_IP = request['REMOTE_ADDR']
-    if Settings.Wired.Changed == 0:
-        Net = _getGadgetMode()
+    Net = _getGadgetMode()
```

Comment:
My idea here was, when turning on a wired device (ethernet dongle, or ethernet gadget) we test it, get the address then go back to the original setting, but make it final in the last tab.
This way you know what will happen (like AP disappears, and wifi appears) which might be needed to find the web server again (ip address can change).

Settings.xx.Changed was intended to capture before (so we can return to it)  and after (so we can make it final in the last tab) state of a change.

Reply:
Okay, we might revert it, but for sure I changed it since with my coding it did not work any more as it supposed to do previously.
For now, all the network settings (Wifi, Bluetooth, Wired) get applied instantanly which has some disadvantages as you correctly state. But I suggest to first merge the PR and then to re-introduce the lazy settings feature for all typologies in a consistent way.

Reply:
Except all kind of stuff is not working now for me. It's probably just a small thing, but I haven't found the reason yet. Working on it.

Reply:
Ping?

Reply:
Hi @mwallnoefer I haven't forgotten. As your patches break about the same as they fix I plan to test them one by one and fix what they break, so that will need some work. So first I need to complete the rather substantial patch set for btrfs (almost done). And then I'll get to this. Sorry for the wait.

Reply:
Two months have passed so I ask you again: any news, @htot?

I think at least the first two commits 3b02931 and 6409ee0 should be fine...